users/tests/user_admin_test.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+
+import users.models
+from users.admin import CustomUserAdmin
+from users.models import User
+
+FieldsetsType = tuple[tuple[None, dict[str, str | tuple[str]]]]
+
+
+def get_registered_model_admin(model: users.models) -> admin.ModelAdmin:
+    model_admin_class = admin.site.get_model_admin(model)
+
+    if model_admin_class is None:
+        message = f"No admin registered for model {model}"
+        raise ValueError(message)
+    return model_admin_class
+
+
+@pytest.fixture()
+def model_admin() -> admin.ModelAdmin:
+    return get_registered_model_admin(User)
+
+
+def test_admin_registered(model_admin: admin.ModelAdmin) -> None:
+    assert isinstance(model_admin, CustomUserAdmin)
+
+
+def test_list_display(model_admin: admin.ModelAdmin) -> None:
+    assert model_admin.list_display == DjangoUserAdmin.list_display[1:]
+
+
+def test_ordering(model_admin: admin.ModelAdmin) -> None:
```

Comment:
This is a good exercise but not a needed test. You are testing what is provided and tested by Django (ordering)